src/routes/auth.route.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const { authController } = require('../controllers/auth.controller.js');
+const { catchError } = require('../middlewares/catchMiddleware.js');
+const { authMiddleware } = require('../middlewares/authMiddleware.js');
+
+const authRouter = new express.Router();
+
+authRouter.post('/registration', catchError(authController.register));
+
+authRouter.get(
+  '/activation/:activationToken',
+  catchError(authController.activate),
+);
+
+authRouter.post('/login', catchError(authController.login));
+authRouter.post('/logout', authMiddleware, catchError(authController.logout));
+authRouter.get('/refresh', authMiddleware, catchError(authController.refresh));
```

Comment:
Consider removing `authMiddleware` from the `/refresh` route. Typically, the refresh token endpoint should not require authentication since it is used to obtain a new access token.